docs/inference_protocols.md

```diff
@@ -47,6 +47,21 @@ model health, metadata and statistics. Additional endpoints allow
 model loading and unloading, and inferencing. See the KFServing and
 extension documentation for details.
 
+### GRPC KeepAlive Options
+
+Triton exposes GRPC KeepAlive parameters with the default values for both
+client and server described [here](https://github.com/grpc/grpc/blob/master/doc/keepalive.md).
+
+You can find a `KeepAliveOptions` struct that encapsulates these parameters
```

Comment:
I think it's not a good idea to point to our internal implementation details in generic user documentation. 
You may list all the exposed cli options here and may be an example server invocation.